unity/CAVE/Assets/W3D_Translator/CLI.cs

```diff
@@ -40,26 +41,26 @@ public static void Main()
             // The path to the xml is send as a command line argument
             // const string xmlPath = GetXmlPathArg();
             const string xmlPath = "../../test/sample.xml";
-            Story xml = LoadStory(xmlPath);
+            LoadStory(xmlPath);
 
             // Create new scene and store the root GameObjects
             // InstantiationResult instantiatedScene = InstantiateScene(xmlPath);
             // GameObject xrRig = instantiatedScene.scene.GetRootGameObjects()[0];
-            // GameObject story = instantiatedScene.scene.GetRootGameObjects()[1];
+            // story = instantiatedScene.scene.GetRootGameObjects()[1];
```

Comment:
Why is there commented out code?

Reply:
You can't call `InstantiateScene` during "play" (runtime), so it only works when I use Unity's CLI (from the Python script). But that adds a _lot_ of overhead with launching Unity every time I want to test a code change.

Instead of creating a new scene I'm just using the base scene and getting the GameObjects in the same way. It makes development a lot quicker - same reason why the path to the xml is hard coded above it